readme-docs/KUBERNETES.md

```diff
@@ -52,7 +52,6 @@ helm repo update
 
```

Comment:
We should  have in documentation what version are supported. All this changes needs to be documented during the release, because migration to bitnami for some users could be problematic.

Reply:
I added a section, but I don't want to write the exact versions in the docs. It will outdated soon because someone will forget to update it.